internal/querynodev2/tasks/fifo_policy.go

```diff
@@ -0,0 +1,48 @@
+package tasks
+
+import (
+	"github.com/milvus-io/milvus/pkg/util/paramtable"
+)
+
+var _ schedulePolicy = &fifoPolicy{}
+
+// newFIFOPolicy create a new fifo schedule policy.
+func newFIFOPolicy() schedulePolicy {
+	return &fifoPolicy{
+		queue: newMergeTaskQueue(""),
+	}
+}
+
+// fifoPolicy is a fifo policy with merge queue.
+type fifoPolicy struct {
+	queue *mergeTaskQueue
+}
+
+// Push add a new task into scheduler, an error will be returned if scheduler reaches some limit.
+func (p *fifoPolicy) Push(task Task) (int, error) {
```

Comment:
As this never returns an error, maybe remove error from the return values?

Reply:
`fifo` implements a `schedulePolicy` interface,
Another `user-task-polling` policy returns error if some limit reached.